packages/calico-mesos.spec

```diff
@@ -31,13 +32,17 @@ mkdir -p -m 0755 %{buildroot}/calico
 install -m 0755 %{SOURCE0} %{buildroot}/calico
 install -m 0755 %{SOURCE2} %{buildroot}/calico
 
+mkdir -p -m 0755 %{buildroot}%{_sysconfdir}/mesos-slave
+install -m 0755 %{SOURCE4} %{buildroot}%{_sysconfdir}/mesos-slave/
+
 ############################################
 %files
 
 /calico/calico_mesos
 /calico/modules.json
-/%{_unitdir}/%{name}.service
-/%{_bindir}/calicoctl
+%{_unitdir}/%{name}.service
+%{_bindir}/calicoctl
+%{_sysconfdir}/mesos-slave/modules
```

Comment:
I've taken advantage of how [mesos-init-wrapper](https://github.com/deric/mesos-deb-packaging/blob/master/mesos-init-wrapper)  loads start flags. The new `/etc/mesos-slave/modules` that you've commented on is a file with contents: `file:///calico/modules.json`. This is interpreted as `mesos-slave --modules=file:///calico/modules.json` when mesos-init-wrapper is called. This allows us to easily add the start flag to mesos from this package, without having to create custom service files, etc.

The actual modules.json file itself is handled elsewhere (line 42 and line 33)